### DISCUSSION TITLE
Set night mode from HA

### DISCUSSION THREAD

**Discussion by nielscil** [revk/xESP32-Faikout#350] [Q&A]
Hi,

tnx for this awesome project. I've got a question about the night-mode. I want my units to always use the quiet/night-mode but, as already said S21 does have some problems with that so it automatically reports auto mode. I'm trying to use automations for it to trigger but I can't seems to get it working every time. Is there a way to force Faikin to always send quiet/night mode?

**Comment by revk** [comment]
Ah funny you should ask.

Latest beta has `new alwaysquiet setting that sets night mode not auto` feature.

Try that.

**Reply by nielscil** [reply]
Nice, will try that!
